### PR TITLE
Show overflow for #8118.

### PR DESCRIPTION
**Pull request by notriddle**
Fixes list items.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/8145)

### CODE REVIEW

File: tests/wpt/mozilla/tests/css/list_item_overflow.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<body>
+<ul style="position: fixed; padding: 0; left: 20px; border: solid 10px rgba(0,0,0,0.1)">
```
Comment (eefriedman) [inline tests/wpt/mozilla/tests/css/list_item_overflow.html]: The "border" attribute isn't really essential to the testcase... I'd suggest getting rid of it.

File: tests/dromaeo/dromaeo [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit 0df04c071f49dce9ecf578c4e152706aacc886f1
```
Comment (eefriedman) [inline tests/dromaeo/dromaeo]: Is this supposed to be part of this PR?

**Comment by eefriedman** [comment]
Otherwise looks fine.

**Comment by jdm** [comment]
How were the manifest changes made? The reftest is missing a `<meta>` attribute specifying the reference file to match again.

**Comment by notriddle** [comment]
I edited the MANIFEST.json file by hand; I suppose there's a better way to
do it, but I couldn't find how.

**Comment by jdm** [comment]
For future reference, `./mach test-wpt --update-manifest` automatically generates appropriate changes, but even easier is `./mach create-wpt --reftest /path/to/file_to_create.html --reference /path/to/reference_file_to_create.html` that uses appropriate templates and ensures MANIFEST.json is updated appropriately.

**Comment by notriddle** [comment]
```
$ ./mach test-wpt --update-manifest
It looks like you passed an unrecognized argument into mach.

The test-wpt command does not accept the arguments: --update-manifest
```

**Comment by jdm** [comment]
Sorry, `--manifest-update`.

**Comment by notriddle** [comment]
r?

**Comment by notriddle** [comment]
I rebased it. Why hasn't @highfive removed the tag?

**Comment by jdm** [comment]
The mergeability status isn't always present in the notification that highfive receives. The docs have been unclear if there's another notification that should be listened for.

**Comment by notriddle** [comment]
Rebased again. r?

**Pull request merged**